cineast-core/src/main/java/org/vitrivr/cineast/core/features/FESVectorFeature.java

```diff
@@ -414,6 +414,7 @@ private float[] embedModel(IModel model) throws IOException, InterruptedExceptio
 
   private float[] embedVideo(List<MultiImage> frames) throws IOException, InterruptedException {
     var images = frames.stream().map(image -> image.getBufferedImage()).collect(Collectors.toList());
+    images = images.stream().map(img -> new BufferedImage(img.getWidth(), img.getHeight(), BufferedImage.TYPE_INT_RGB)).collect(Collectors.toList());
```

Comment:
You are overwriting the previously generated images with empty images here, that is almost certainly not what you want.